src/Network/Packets.cs

```diff
@@ -879,23 +879,25 @@ public PBulletinBoardRequestMessageSummary(uint serial, uint msgserial) : base(0
 
     internal sealed class PBulletinBoardPostMessage : PacketWriter
     {
-        public PBulletinBoardPostMessage(uint serial, uint msgserial, string subject, string message) : base(0x71)
+        public PBulletinBoardPostMessage(uint serial, uint msgserial, string subject, Game.UI.Controls.MultiLineBox _textBox) :
```

Comment:
Instead of use MultilineBox to take the text, just pass the Text as string.
The reason is code consistency. Packet needs a string and you use the .Text property only from MultiLineBox.

Basically rollback the sign.